data-raw/babynames_sample.csv

```diff
@@ -0,0 +1,55 @@
+year,sex,name,n,prop
+1898,F,Catherine,1705,0.006219359168903934
```

Comment:
Why do we end up with so many significant digits here?

Reply:
This is the default behavior I'm getting with `readr_1.2.1`.  data-raw/births.R has the entire script.  Whatever is happening, write.csv makes a similar decision regarding the number of digits to display.  Note this is a .csv and not a printed tbl.